src/global_stats.h

```diff
@@ -39,6 +39,7 @@ typedef struct {
   size_t total_query_commands;        // Number of successful query commands, including `FT.CURSOR READ`
   clock_t total_query_execution_time; // Total time spent on queries (in clock ticks)
   uint_least8_t used_dialects;        // bitarray of dialects used by all indices
+  size_t logically_deleted;           // Number of logically deleted documents in all indices
```

Comment:
please add that `logically_deleted` means documents that were marked wit `DELETED` flag but whose memory was not yet cleaned by the GC